docs/02-app/01-building-your-application/04-caching/index.mdx

```diff
@@ -0,0 +1,468 @@
+
+When fetching data on the server, a `fetch` request first checks the Data Cache for a cached response. If a cached response is found, it is returned immediately. If not, the request is made to the data source, and the response is cached.
+
+> **Static vs. Dynamic Data Fetching**:
+>
+> You may see the terms **Static Data Fetching** and **Dynamic Data Fetching** being used. Static Data Fetching refers to cached data requests (default), whereas Dynamic Data Fetching refers to data that is fetched on every request, at runtime.
+
+#### Duration
+
+The Data Cache is persistent across user requests and deployments unless you revalidate or opt-out.
+
+#### Revalidating
+
+Revalidation is the process of purging the Data Cache and re-fetching the latest data. This is useful when your data changes and you want to ensure you show the latest information.
+
+Cached data can be revalidated in two ways:
+
+- **Time-based revalidation**: Automatically revalidate data after a certain amount of time has passed. This is useful for data that changes infrequently and freshness is not as critical. Time-based revalidation can be configured with the [Route Segment Config options](#segment-config-options) or per individual [`fetch` request](#fetchoptionsnextrevalidate).
+- **On-demand revalidation**: Manually revalidate data based on an event (e.g. form submission). On-demand revalidation can use a tag-based or path-based approach to revalidate groups of data at once. This is useful for when you want to ensure the latest data is always shown. On-demand revalidation can be configured with the [`revalidateTag`](#fetchoptionsnexttag-and-revalidatetag) and [`revalidatePath`](#revalidatepath) APIs.
+
+> **Good to know**: Check if your upstream data provider has caching enabled by default. You might need to disable (e.g. `useCdn: false`), otherwise a revalidation won't be able to pull fresh data to update the Data Cache. Caching can occur at a CDN (for an endpoint being requested) when it returns the `Cache-Control` header. ISR on Vercel [persists the cache globally and handles rollbacks](https://vercel.com/docs/concepts/incremental-static-regeneration/overview).
+
+#### Opting out
+
+For individual data fetches, you can opt out of caching by setting the [`cache`](#fetchoptionscache) option to `no-store`. This means data will be fetched dynamically, on each request.
+
+```jsx
+// Opt out of caching
+fetch(`https://...`, { cache: 'no-store' })
+```
+
+Alternatively, you can also use the [Route Segment Config options](#segment-config-options) to opt out of caching for **all** data requests in a route segment.
```

Comment:
Maybe show an example of what this looks like?

Reply:
Yes, will do!